price-estimator/src/main.rs

```diff
@@ -158,7 +158,14 @@ fn main() {
         options.orderbook_update_interval,
     ));
 
-    let filter = filter::all(orderbook, token_info).with(warp::log("price_estimator"));
+    // We add the allow origin header so that requests from the interactive openapi documentation
+    // go through to locally running instance.
```

Comment:
Is this still up to date? It seems like we don't need this for openapi anymore.

Reply:
It is needed or the interactive docs are unable to access the locally running price estimator, even if the interactive docs are also running from localhost.